lib/swaps/Swaps.ts

```diff
@@ -759,6 +759,13 @@ class Swaps extends EventEmitter {
     try {
```

Comment:
nit: I'm wondering if it's worth extracting this try/catch block into `sendMakerPayment(swapClient, deal)` and unit test the catch block behaviour?

Reply:
It's a fine idea, but I just took a stab at this and it involved quite a bit of mocking. Perhaps in another PR? Or if you have an approach in mind maybe you can take a stab at it. It would be good to have a test for the `PAYMENT_REJECTED` error as well.